astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -0,0 +1,402 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import urllib.request
+import os
+from datetime import datetime, timedelta
+
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from astropy import _erfa as erfa
+from astropy.time import Time
+from astropy.utils.iers import iers
+from astropy.utils.data import get_pkg_data_filename
+from astropy.tests.helper import catch_warnings
+
+
+SYSTEM_FILE = '/usr/share/zoneinfo/leap-seconds.list'
```

Comment:
Is this path hardcoding intentional?

Reply:
If it is, that is a pretty strict one. Definitely won't work on Windows or for any Unix users that don't or can't use `/usr/share`.

Reply:
I think this is OK as it stands because it only is valid on `linux` platforms since windows doesn't have as standard of a location, and we get linux/mac on CI anyway.

(where's it's used it skips the test if the file isn't present)

Reply:
Although it *might* be better to do a skipif windows trick instead? But I think that shouldn't hold up the PR otherwise

Reply:
I really hope to get system files from other distributions/systems, but let that be follow-up as well - see #9479